code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -1757,3 +1757,402 @@
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/shoyu_ramen
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/shoyu_ramen
+	name = "shōyu ramen"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "shoyu_ramen"
+	drink_type = MEAT | GRAIN | VEGETABLES | SEAFOOD
+
+/datum/chemical_reaction/food/soup/shoyu_ramen
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 20,
+		/datum/reagent/consumable/nutriment/soup/teriyaki = 15,
+	)
+	required_ingredients = list(
+		/obj/item/food/spaghetti/boilednoodles = 1,
+		/obj/item/food/kamaboko_slice = 1,
+		/obj/item/food/meat/cutlet = 1,
+		/obj/item/food/boiledegg = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/shoyu_ramen = 30,
+		/datum/reagent/consumable/nutriment/vitamin = 8,
+		/datum/reagent/consumable/nutriment/protein = 8,
+	)
+
+// Gyuramen
+/datum/reagent/consumable/nutriment/soup/gyuramen
+	name = "Gyuramen Miy Käzu"
+	description = "A rich beef and onion ramen with cheese- blending several cultural influences seemlessly into one tasty dish."
+	data = list("beef broth" = 1, "onion" = 1, "cheese" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/gyuramen
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/gyuramen
+	name = "gyuramen miy käzu"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "gyuramen"
+	drink_type = MEAT | GRAIN | DAIRY | VEGETABLES
+
+/datum/chemical_reaction/food/soup/gyuramen
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 20,
+		/datum/reagent/consumable/soysauce = 5,
+	)
+	required_ingredients = list(
+		/obj/item/food/spaghetti/boilednoodles = 1,
+		/obj/item/food/cheese/wedge = 1,
+		/obj/item/food/onion_slice = 2,
+		/obj/item/food/meat/cutlet = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/gyuramen = 30,
+		/datum/reagent/consumable/nutriment/vitamin = 2,
+		/datum/reagent/consumable/nutriment/protein = 10,
+	)
+
+// New Osaka Sunrise
+/datum/reagent/consumable/nutriment/soup/new_osaka_sunrise
+	name = "New Osaka Sunrise Soup"
+	description = "A bright, flavourful miso soup with tofu that commonly forms part of a traditional Martian breakfast, at least in the capital."
+	data = list("miso" = 1, "tofu" = 1, "onion" = 1, "eggplant" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/new_osaka_sunrise
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/new_osaka_sunrise
+	name = "\improper New Osaka Sunrise soup"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "new_osaka_sunrise"
+	drink_type = MEAT | GRAIN | DAIRY | VEGETABLES
+
+/datum/chemical_reaction/food/soup/new_osaka_sunrise
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/miso = 15,
+	)
+	required_ingredients = list(
+		/obj/item/food/grown/herbs = 1,
+		/obj/item/food/grown/eggplant = 1,
+		/obj/item/food/onion_slice = 1,
+		/obj/item/food/tofu = 1,
+	)
```

Comment:
This recipe may want `percentage_of_nutriment_converted = 0.15` or even lower, very potent plants may end up overpowering it.